### ISSUE TITLE
[Feature] Activate the Machine on the first waypoint if you select that

### ISSUE THREAD

**Issue by Zwox89** [Courseplay/Courseplay_FS22#402]
**What is the problem this feature/enhancement solves?**
When you create a course and select "Start on first waypoint" and you leave your vehicle somewhere else on the field or even outside the field, the fertilizer spreader starts spreading.

**Describe the solution you'd like**
1. Don't start the machine outside of the field
2. If you select "Start on first waypoint" activate the machine there and not way before.

**Comment by Tensuko** [comment]
And you use wich Version ?!

**Comment by Zwox89** [comment]
v7.0.0.16

Edit: I think the problem is with the "Arrow" that you have to place on the Map. If that arrow gets reached, he starts the machine, even if the "Arrow" is in the middle of the Field. So it's probably game side

**Comment by RealNeo80** [comment]
i`ve seen this in other versions also if you say first waypoint and you have a spreader or sprayer it switches on immediatly (sprayer directly after it is unfolded) even if they haven´t reacht the first waypoint.

![image](https://user-images.githubusercontent.com/8066324/148586620-82cd497f-d5bb-496e-a112-bab7450d963b.png)

**Comment by sirbum69** [comment]
yes i had this happen yesterday also. It started spraying right away. Also does the cultivator the same, will start it way off the field.

this is on the latest install also.

**Comment by RealNeo80** [comment]
#259 i think it had something to do with my other request ... or at least a bit related ;)

**Comment by Tensuko** [comment]
@RealNeo80 yes, thats somehow related :)
